config_files/tele_config_hera_dipole_cubic.yaml

```diff
@@ -0,0 +1,8 @@
+beam_paths:
+  0: '/ilifu/astro/projects/hera/Validation/h4c_sim/config_files/NF_HERA_Dipole_efield_beam.fits'
```

Comment:
this is not computer-agnostic. I recommend having a little script that 'makes' the tele config, and knows the path to the beam file for that computer. Makes it easier to set the frequency/spatial interpolation as well.

Reply:
sounds good. maybe another  file call h4c_sim_info.yaml or something like that

Reply:
Yeah, something like that.